data/multiplayer/scenarios/4p_King_of_the_Hill.cfg

```diff
@@ -54,9 +48,135 @@
         side=4
         team_name=north-west
         user_team_name= _ "teamname^Northwest"
-        canrecruit=yes
         controller=human
         gold=150
-        fog=yes
     [/side]
+
+    [event]
+        name=prestart
+        [objectives]
+            [objective]
+                description= _ "Defeat enemy leaders"
+                condition=win
+            [/objective]
+            [note]
+                description= _ "A leader reaching the central castle keep destroys all other castles"
+            [/note]
+            [note]
+                description= _ "All castles are restored if the central keep is abandoned"
+            [/note]
+            [note]
+                description= _ "Unallied computer players will temporarily unite against a leader on the central keep"
+            [/note]
+        [/objectives]
+        [lift_fog]
+            x,y=18,18
+            radius=4
+            multiturn=yes
+        [/lift_fog]
+        [remove_shroud]
+            x,y=18,18
+            radius=4
+        [/remove_shroud]
+    [/event]
+
+    # Leaders must not move on first turn ahead of recruits, to balance team games against an opening leader rush for the Hill.
+    [event]
+        name=turn 1
+        [modify_unit]
+            [filter]
+                canrecruit=yes
+            [/filter]
+            moves=0
```

Comment:
I dont think this does what you think it does

Reply:
it kind of does. I tested the addon version of this and it works as intended

Reply:
Right, first turn is special and doesnt have refresh.

Reply:
This code was no longer useful after removing the castle destruction mechanic so it is gone now, anyway.